lib/Mojolicious/Guides/Tutorial.pod

```diff
@@ -0,0 +1,938 @@
+
+=encoding utf8
+
+=head1 NAME
+
+Mojolicious::Guides::Tutorial - Get started with Mojolicious
+
+=head1 TUTORIAL
+
+A quick example-driven introduction to the wonders of L<Mojolicious::Lite>. Almost everything you'll learn here also
+applies to full L<Mojolicious> applications.
+
+This is only the first of the L<Mojolicious::Guides>. Other guides delve deeper into topics like
+L<growing|Mojolicious::Guides::Growing> a L<Mojolicious::Lite> prototype into a well-structured L<Mojolicious>
+application, L<routing|Mojolicious::Guides::Routing>, L<rendering|Mojolicious::Guides::Rendering> and more. It is
+highly encouraged that readers continue on to the remaining guides after reading this one.
+
+=head2 Hello World
+
+A simple Hello World application can look like this, L<strict>, L<warnings>, L<utf8> and Perl 5.16 L<features|feature>
+are automatically enabled and a few L<functions|Mojolicious::Lite/"FUNCTIONS"> imported, when you use
+L<Mojolicious::Lite>, turning your script into a full featured web application.
+
+  #!/usr/bin/env perl
+  use Mojolicious::Lite -signatures;
+
+  get '/' => sub ($c) {
+    $c->render(text => 'Hello World!');
+  };
+
+  app->start;
+
+With L<Mojolicious::Command::Author::generate::lite_app> there is also a helper command to generate a small example
+application.
+
+  $ mojo generate lite-app myapp.pl
+
+=head2 Commands
+
+Many different L<commands|Mojolicious::Commands/"COMMANDS"> are automatically available from the command line. CGI and
+L<PSGI> environments can even be detected and will usually just work without commands.
+
+  $ ./myapp.pl daemon
+  Web application available at http://127.0.0.1:3000
+
+  $ ./myapp.pl daemon -l http://*:8080
+  Web application available at http://127.0.0.1:8080
+
+  $ ./myapp.pl cgi
+  ...CGI output...
+
+  $ ./myapp.pl get /
+  Hello World!
+
+  $ ./myapp.pl
+  ...List of available commands (or automatically detected environment)...
+
+A call to L<Mojolicious/"start"> (C<app-E<gt>start>), which starts the command system, should be the last expression in
+your application, because its return value can be significant.
```

Comment:
In what way is the return value significant?  Also, shouldn't this be last because it starts the loop and any code after it will never be part of the app?